core/src/main/proto/spotify_100.proto

```diff
@@ -20,6 +20,9 @@ message Metric {
 
     // resource "tags" associated to metric.
     map<string, string> resource = 5;
+
+    // Point value
+     Point point =  10;
```

Comment:
I don't think Point is relevant here. This `Metric` has new type of value. The `time` you have in `Point` already available in `Metric`.
I think you should use same approach as [here](https://github.com/spotify/ffwd-client-java/blob/master/ffwd-client/src/main/proto/protocol1.proto)

Reply:
Also why field number 10? I think the next number should be 6 if we introducing new field to proto.

Reply:
Dropwizard metrics are still sending value in double format. We are using the new definition for distribution only. So I cannot just change double into object.  
Because it is a opt- in feature.

Reply:
Double value will be sent by old FFWD users and Value value will be sent by new FFWD users. What Point object represents what is not described already? Metric is bunch of tags and value. Old value is double, new value is Value that's what should be add as in here https://github.com/spotify/ffwd-client-java/blob/master/ffwd-client/src/main/proto/protocol1.proto

Reply:
Let setup a meeting and chat about this offline.  Unless we want to ask Heroic users to send new metric we cannot change this.

Reply:
As discussed offline, we cannot just replace the value format.   Also we cannot use oneof in this case. Please look at this as 2 versions in one file.  If you don't like the idea of sending new version  value wrap into a Point object. I will remove it. 
I think this conversation could have been avoided if we have a single global definition of metric the current state can easily lead to confusion and bugs.

Reply:
Hey @ao2017 it's not that I don't like this solution - I think that Point is useless and we could describe what we want with `Value newValue`. Let's remove Point as we agreed offline. Thanks